frontend/src/pages/team/Instances.vue

```diff
@@ -3,21 +3,22 @@
         <template #header>
             <ff-page-header title="Instances">
                 <template #context>
-                    A list of all Node-RED instances belonging to this Team.
+                    <span v-if="!dashboardRoleOnly">A list of all Node-RED instances belonging to this Team.</span>
+                    <span v-else>A list of Node-RED instances with Dashboards belonging to this Team.</span>
```

Comment:
Niggle: 

The if `variable` is unrelated to the text. Feels wrong. Can we reverse the logic for clearer relationship?

Also, related to the suggestion about showing <b>Dashboards</b> it might be clearer/simpler to state "A list of Node-RED Dashboards in this Team."

```suggestion
                    <span v-if="dashboardRoleOnly">A list of Node-RED Dashboards in this Team.</span>
                    <span v-else>A list of all Node-RED instances belonging to this Team.</span>
```